extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -70,19 +71,26 @@ $logo-url: $logo-url-light;
 
 	.entry-point-new {
 		background: $color-buttons;
-		border-radius: 8px;
 		color: $color-button-text;
 		font-size: 10px;
-		height: 18px;
-		line-height: 18px;
-		padding: 0 10px;
+		height: 17px;
+		left: 0;
+		line-height: 17px;
+		position: absolute;
 		text-align: center;
 		text-transform: uppercase;
+		top: 0;
+		width: 41px;
+	}
+
+	.entry-point-description-top {
+		font-size: 14px;
```

Comment:
font size 14px is defined for parent already so I think is should be safe to skip it